core/controllers/payload_validator.py

```diff
@@ -74,6 +89,27 @@ def validate(handler_args, handler_args_schemas, allowed_extra_args):
     return normalized_value, errors
 
 
+def convert_string_to_bool(param):
+    # type: (Text) -> bool
+
+    """Converts a request param of type string into expected bool type.
+
+    Args:
+        param: str. The params which needs normalization.
+
+    Returns:
+        bool. Converts the string param into its expected bool type.
+    """
+    case_insensitive_param = param.lower()
+
+    if case_insensitive_param == 'true':
+        return True
+    elif case_insensitive_param == 'false':
+        return False
+    else:
+        return False
```

Comment:
Should we throw an error here if something else is received?

Reply:
Added a comment, please take a look.